manifests/pipelines/aiedge-e2e.pipeline.yaml

```
@@ -136,7 +136,22 @@ spec:
     - input: "$(params.fetch-model)"
       operator: in
       values: ["git"]
-  # Use kserve-download-model container image to fetch from S3 storage
+  - name: cleanup-git-fetch
+    params:
+    - name: model-name
+      value: $(params.model-name)
+    - name: subdirectory
+      value: model_dir-$(params.model-name)
+    - name: src-model-relative-path
+      value: pipelines/models
```

This should be
```suggestion
    - name: src-model-relative-path
      value: $(params.model-relative-path)
```
right?
Not hard-coded.

---

Yep my mistake